ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -536,4 +538,226 @@ public void getCompletedJobs() throws IOException, SignatureException, XmlProces
 
         assertThat(getCompletedJobsResponse.getStatusCode(), is(200));
     }
+
+    @Test
+    public void clearCompletedJobs() throws IOException, SignatureException, XmlProcessingException {
+        final ClearAllCompletedJobsSpectraS3Response clearAllCompletedJobsResponse = client
+                .clearAllCompletedJobsSpectraS3(new ClearAllCompletedJobsSpectraS3Request());
+
+        assertThat(clearAllCompletedJobsResponse.getStatusCode(), is(204));
+    }
+
+    @Test
+    public void putJobCreatedNotification() throws IOException, SignatureException {
+        UUID notificationUUID = null;
+        try {
+            final PutJobCreatedNotificationRegistrationSpectraS3Response response = client
+                    .putJobCreatedNotificationRegistrationSpectraS3(new PutJobCreatedNotificationRegistrationSpectraS3Request("test@test.test"));
+            notificationUUID = response.getJobCreatedNotificationRegistrationResult().getId();
+            assertThat(response.getStatusCode(), is(201));
+
+        } finally {
+            assertNotNull(notificationUUID);
```

Comment:
I tried it, and yeah, asserts or null pointer exceptions in the "finally" get reported and the main response code assert is not reported.  I can move the assertNotNull up to the try block, but then I don't think it will execute if the response code fails.  Maybe I should to an if (!null) delete the notification instead of an assert?